### PR TITLE
DOCSP-44873 1.44.6 Release Notes

### PR DESCRIPTION
**Pull request by kennethdyer**
## DESCRIPTION

Release notes 1.44.6.

## STAGING
* [RN](https://deploy-preview-691--docs-compass.netlify.app/release-notes/#compass-1.44.6)

## JIRA
* [DOCSP-44873](https://jira.mongodb.org/browse/DOCSP-44873)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,22 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.44.6
+----------------
+
+*Released October 31, 2024*
+
+- Increase the character size restriction for query and
+  aggregation genai :issue:`COMPASS-8369`)
```
Comment (jmd-mongo) [inline source/release-notes.txt:19]: [typo]

Missing open parens on line 19.

**Review by jmd-mongo — CHANGES_REQUESTED** [review]
Thanks for these release notes, @kennethdyer! I noted a small typo.

Thanks,
Joe

File: source/release-notes.txt (hunk 1 above)
Comment (kennethdyer) [reply source/release-notes.txt]: Good catch. Fixed.

**Review by jmd-mongo — APPROVED** [review]
Thanks, @kennethdyer! This one LGTM.

Thanks,
Joe

**Review by paula-stacho — APPROVED** [review]

**Pull request merged**